<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGuideLines</class>
 <widget class="QDialog" name="CreateGuideLines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Guide Lines</string>
  </property>
  <widget class="QComboBox" name="AlignmentCB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="GLGroupCB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SelecetAlignmentL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Alignment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="SelectGLGroupL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Guide Lines Group:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LeftLengthLE">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RightLengthLE">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="LeftLengthL">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Line Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="RightLengthL">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Line Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="TIncrementL">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>260</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Increment Along Tangents:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TIncrementLE">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelB">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddGLGroupB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="CSIncrementL">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>290</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Increment Along Curves-Spirals:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CSIncrementLE">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="EndStationL">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>End Station:</string>
   </property>
  </widget>
  <widget class="QLabel" name="StartStationL">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Station:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EndStationLE">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="StartStationLE">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="FromAlgStartChB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>From Alignment Start</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="ToAlgEndChB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>To Alignment End</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="HorGeoPointsChB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>At Horizontal Geometry Points</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
